types/mempool/priority_nonce.go

```diff
@@ -29,6 +29,7 @@ type priorityNonceMempool struct {
 	senderIndices  map[string]*skiplist.SkipList
 	scores         map[txMeta]txMeta
 	onRead         func(tx sdk.Tx)
+	txReplacement  func(oldTxPriority, newTxPriority int64) bool
```

Comment:
Can `func(oldTxPriority, newTxPriority int64) bool` be defined by a type alias and that be used instead? The signature is a bit long to read.

Reply:
updated